src/nft/components/profile/list/PriceTextInput.tsx

```diff
@@ -132,7 +144,7 @@ export const PriceTextInput = ({
 
   return (
     <PriceTextInputWrapper>
-      <InputWrapper borderColor={borderColor}>
+      <InputWrapper borderColor={warningColor}>
```

Comment:
might be worth typing `warningColor`

Reply:
It's currently typed as a string based off potential values
<img width="287" alt="Screen Shot 2023-02-06 at 11 39 09 " src="https://user-images.githubusercontent.com/11512321/217068510-b8410a48-539d-4b95-a6ae-4550bb5fcc94.png">